scripts/check-js-api-version.js

```diff
@@ -0,0 +1,24 @@
+// Protractor configuration file, see link for more information
+// https://github.com/angular/protractor/blob/master/lib/config.ts
+
+require('dotenv').config({path: process.env.ENV_FILE});
+const fs = require('fs');
+
+const configPath = './dist/demo-shell/app.config.json';
+
+fs.readFile(configPath, (err, appConfigString) => {
```

Comment:
Is it going to wait before ending the script?? Should not it be readFileSync, or some Promise/async-await magic?